.github/workflows/ci.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: lxd
-          bootstrap-options: "--agent-version 2.9.29"
+          bootstrap-options: "--agent-version 2.9.38"
```

Comment:
AFAIK, `actions-operator` installs the latest Juju 2.9 LTS (which is 2.9.38), so technically you can remove `bootstrap-options` and follow the latest juju.